pkg/api/v1/types.go

```diff
@@ -704,6 +704,11 @@ type Container struct {
 	ImagePullPolicy PullPolicy `json:"imagePullPolicy,omitempty" description:"image pull policy; one of Always, Never, IfNotPresent; defaults to Always if :latest tag is specified, or IfNotPresent otherwise; cannot be updated; see http://releases.k8s.io/HEAD/docs/user-guide/images.md#updating-images"`
 	// Optional: SecurityContext defines the security options the pod should be run with
 	SecurityContext *SecurityContext `json:"securityContext,omitempty" description:"security options the pod should run with; see http://releases.k8s.io/HEAD/docs/design/security_context.md"`
+
+	// Variables for interactive containers, these have very specialized use-cases (e.g. debugging)
+	// and shouldn't be used for general purpose containers.
+	TTY   bool `json:"tty,omitempty" description:"Should this container allocate a TTY for itself, also requires 'stdin' to be true"`
+	StdIn bool `json:"stdin,omitempty" description:"Should this container allocate a buffer for stdin in the container runtime"`
```

Comment:
Field should match other instances in the API. Consistency, consistency, consistency.